doc/cabal-commands.rst

```diff
@@ -832,24 +832,24 @@ configuration from the 'cabal.project', 'cabal.project.local' and other files.
 
     ::
 
-        $ cabal repl --build-depends "vector >= 0.12 && < 0.13"
+        $ cabal repl --build-depends="vector >= 0.12 && < 0.13"
 
     Both of these commands do the same thing as the above, but only expose ``base``,
     ``vector``, and the ``vector`` package's transitive dependencies even if the user
     is in a project context.
 
     ::
 
-        $ cabal repl --ignore-project --build-depends "vector >= 0.12 && < 0.13"
-        $ cabal repl --project='' --build-depends "vector >= 0.12 && < 0.13"
+        $ cabal repl --ignore-project --build-depends="vector >= 0.12 && < 0.13"
```

Comment:
Does it not work with just a space? I thought it does, and I actually use space more often because it's usually easier to reach than `=`. So, I'm not sure this is that good of a change. 

Can we say somewhere that space is admissible in the long form? Also, in `-b DEPENDENCIES or -bDEPENDENCIES` I'd leave just one and say somewhere that the other is admissible.

Reply:
The `--build-depends` does work without the space but the `--jobs` of `cabal build all --jobs 1` doesn't so to be on the safe side I used `=` for long options. Are there any long options that don't work without `=` where some people might use ` `?